backend/app/models/study_response_export.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'digest/md5'
+
+class StudyResponseExport < ApplicationRecord
+  belongs_to :study
+
+  def self.new_random_seed
+    Random.new_seed
+  end
+
+  scope :pending, -> { where(is_complete: false) }
+
+  has_many_attached :files
+
+  def fresh?
+    is_complete && updated_at.after?(4.hours.ago)
+  end
+
+  def fetch
+    return if is_complete?
+    # TODO: kickoff and monitor Qualtrics download
```

Comment:
lets create a followup ticket before/when we merge

Reply:
https://github.com/openstax/research/issues/318